build-go.sh

```diff
@@ -156,7 +175,7 @@ function printInitialDistfile() {
 	jq ".name = \"$distname\"" |
 	jq ".owner  = \"`cat repo-owner`\"" |
 	jq ".description = \"`cat description`\"" |
-	jq ".date = \"`date '+%B %d, %Y'`\"" |
+	jq ".date = \"`date '+%b %d, %y'`\"" |
```

Comment:
Why the lower case `%y`? I'd prefer `date '+%b %d, %Y'`

Reply:
Thanks for catching that! I probably just lowercased the line.